openshift/kustomize/services/capture/base/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ kind: Kustomization
 resources:
   - pvc.yaml
   - config-map.yaml
+  - services-config-map.yaml
```

Comment:
Adding shared config to all services.  Would like to come up with a way to not have to duplicate this file.